Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -50,6 +50,14 @@ - (void)tearDown {
   [super tearDown];
 }
 
+- (void)assertVector:(std::vector<FSTMutationBatch *>)actual
```

Comment:
This is test code, so we don't really care about performance, but I'll point out that you can avoid copying these vectors by passing a const&, i.e.:

```
- (void)assertVector:(const std::vector<FSTMutationBatch *>&)actual
```

Optional.

Reply:
good call, done.